engine/src/operators/GroupBy.cpp

```diff
@@ -9,6 +9,7 @@
 #include "distribution/primitives.h"
 #include "utilities/CommonOperations.h"
 #include "utilities/RalColumn.h"
+#include "utilities/DebuggingUtils.h"
```

Comment:
im guessing this is not meant to be in here either

Reply:
removed thanks